TS: Error in type declaration

Hi I have an issue when using the package with typescript : 

```
node_modules/js-video-url-parser/lib/index.d.ts:9:15 - error TS2307: Cannot find module './provider/template' or its corresponding type declarations.
```

It looks like the `template` files are not include in the `lib` folder of npm package : 
```
├── base.d.ts
├── base.js
├── index.d.ts
├── index.js
├── provider
│   ├── base-provider.d.ts
│   ├── canalplus.d.ts
│   ├── canalplus.js
│   ├── coub.d.ts
│   ├── coub.js
│   ├── dailymotion.d.ts
│   ├── dailymotion.js
│   ├── soundcloud.d.ts
│   ├── soundcloud.js
│   ├── teachertube.d.ts
│   ├── teachertube.js
│   ├── ted.d.ts
│   ├── ted.js
│   ├── tiktok.d.ts
│   ├── tiktok.js
│   ├── twitch.d.ts
│   ├── twitch.js
│   ├── vimeo.d.ts
│   ├── vimeo.js
│   ├── wistia.d.ts
│   ├── wistia.js
│   ├── youku.d.ts
│   ├── youku.js
│   ├── youtube.d.ts
│   └── youtube.js
├── testUrls.js
├── urlParser.d.ts
├── urlParser.js
└── util.js
```

(:point_up: `node_modules/js-video-url-parser/lib`)

The issue still exists in last release `0.4.3`...

It seems the PR didn't fix it : 
```
node_modules/js-video-url-parser/lib/index.d.ts:9:15 - error TS2307: Cannot find module './provider/template' or its corresponding type declarations.

9 export * from './provider/template';
                ~~~~~~~~~~~~~~~~~~~~~
```

I didn't take a look to the PR before merge, but the problem does not concern this line : 
https://github.com/brunolazzaro/jsVideoUrlParser/blob/c10f242c8ae4299eb89d7ae5013b0efd91f5c37c/lib/provider/template.d.ts#L1

But the import of `template.d.ts` it self wich is not present in release package.

Following @claabs comment, do you want me to make a PR to remove the import line ?

--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -6,7 +6,6 @@ export * from './provider/coub';
 export * from './provider/dailymotion';
 export * from './provider/soundcloud';
 export * from './provider/teachertube';
-export * from './provider/template';
 export * from './provider/twitch';
 export * from './provider/vimeo';
 export * from './provider/wistia';

